tools/mo/openvino/tools/mo/main.py

```diff
@@ -28,71 +29,72 @@
 
 
 def main(cli_parser: argparse.ArgumentParser, framework=None):
-    # Initialize logger with 'ERROR' as default level to be able to form nice messages
-    # before arg parser deliver log_level requested by user
-    init_logger('ERROR', False)
+    with TempFilesCleaner():
```

Comment:
What is the problem which was here? Why do you need to introduce a custom controller?

Reply:
It works same as `try/finally`, but it requires less tabulation changes and less control which scope is covered by `try/finally`

Reply:
Deleted at all